### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by DammyShittu**
## In this PR, I;
- initialized the app with `create-react-app`,
- added React Redux by running `npm install react-redux`,
- structured the application files using a *feature folder* approach and used the *ducks pattern* for the Redux files,
- added `BookInput` component to add new book,
- added React Router, and
- set two (Books and Categories) `Route`s and `Link`s for the app's navigation and added.

Linter files were correctly set up as well.

### CODE REVIEW

File: src/redux/Books/Books.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Books = () => (
+    <div>
+      <h1>All Books</h1>
+    </div>
+  )
+
+export default Books;
```
Comment (arslanbisharat) [inline src/redux/Books/Books.js]: - [ ] Let's add a form as requested in the project instructions. 

Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

**Comment by DammyShittu** [comment]
Hey @arslanbisharat 
Thank you for the review.
I have implemented the requested changes. I will proceed to request for another review now.
Cheers 🚀

**Review by EricMbouwe — APPROVED** [review]
Hi @DammyShittu, 

# Status: Approved✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DammyShittu** [comment]
Thank you @EricMbouwe

**Pull request merged**